content/source/docs/enterprise/saml/identity-provider-configuration-onelogin.html.md

```diff
@@ -9,19 +9,19 @@ sidebar_current: "docs-enterprise2-private-saml-identity-provider-configuration-
 Follow these steps to configure OneLogin as the identity provider (IdP) for Terraform Enterprise.
 
 1. Add a OneLogin app by going to Apps > Add Apps then searching for "SAML Test Connector (IdP)".
-2. Name the app.
+2. In the "Info" tab, enter "Terraform Enterprise" in the "Display Name" field.
```

Comment:
I don't think we want to tell users what to call the app, as they may have their own convention.